src/main/fc/controlrate_profile.h

```diff
@@ -26,6 +26,9 @@
 
 #define CONTROL_RATE_PROFILE_COUNT  6
 
+#define CONTROL_RATE_LIMIT_MIN	200
+#define CONTROL_RATE_LIMIT_MAX	1998
```

Comment:
I preferred if this was dropped, and `SETPOINT_RATE_LIMIT` used, since this is the hard limit we are bound to by the overflow detection. Alternatively we need to throw a build error if this exceeds `SETPOINT_RATE_LIMIT`.

Reply:
My preference will always be a single #define for the hard limit and reference it everywhere. The issue is that it's been put in a source file.
My latest commit addresses that by referencing `#define CONTROL_RATE_CONFIG_RATE_LIMIT_MAX 1998` from `\fc\rc_controls.h` everywhere but maybe that's not quite right either.
Simple question, where should we `#define SETPOINT_RATE_LIMIT 1998` for global use?